bears/installation_tool.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import importlib
+import re
+import subprocess
+import sys
+
+from coala_utils.Question import ask_question
+from coalib.collecting.Collectors import get_all_bears_names
+from coalib.collecting.Importers import iimport_objects
+from coalib.misc.Shell import call_without_output
+from prompt_toolkit import prompt
+from prompt_toolkit.contrib.completers import WordCompleter
+from pygments.lexers import SqlLexer
+
+
+def install_a_pip_package(package_name):
+    """
+    Uses call to install a PyPi package.
+
+    :param package_name: The package to be installed.
+    """
+    subprocess.call(['sudo', 'pip3', 'install', package_name, '--upgrade'])
+
+
+def get_output(command):
+    """
+    Runs the command and decodes the output and returns it.
+
+    :param command: The command to be run.
+    :return:        The output of the command, decoded.
+    """
+    function = subprocess.Popen(command, stdout=subprocess.PIPE)
+    output, err = function.communicate()
+    return output.decode("utf-8")
+
+
+def create_installation_parser():
+    """
+    Creates a parser for command line arguments.
+
+    :return: Parser arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description='Install bears requirements.')
+    parser.add_argument('-i', '--install',
+                        help='Install the requirements',
+                        action='store_true')
+    return parser
+
+
+def main():
+    args = create_installation_parser().parse_args()
+
+    sql_completer = WordCompleter(['All', 'Some', 'None'], ignore_case=True)
```

Comment:
I have no idea, i followed a tutorial and this was the default name, im going to change it ^^